datafusion/core/tests/physical_optimizer/enforce_sorting.rs

```diff
@@ -1652,94 +1647,6 @@ async fn test_not_replaced_with_partial_sort_for_bounded_input() -> Result<()> {
     Ok(())
 }
 
-/// Runs the sort enforcement optimizer and asserts the plan
```

Comment:
I just moved 1200 lines of test code into a different module. I did not change any of the test logic